discord/ext/bridge/core.py

```diff
@@ -376,6 +386,47 @@ def decorator(callback):
     return decorator
 
 
+def guild_only():
+    """Intended to work with :class:`.ApplicationCommand` and :class:`BridgeCommand`, adds a :func:`~ext.commands.check`
+    that locks the command to only run in guilds, and also registers the command as guild only client-side (on discord).
+
+    Basically a utility function that wraps both :func:`discord.ext.commands.guild_only` and :func:`discord.commands.guild_only`.
+    """
+    def predicate(callback: Callable):
+        callback.__guild_only__ = True
```

Comment:
Shouldn't this check if the passed callback is an instance of `ApplicationCommand` and in that case change the `guild_only` attr instead (without underscores)?
Same for `default_member_permissions`.